Client/src/Components/Profile/ProfileForm.tsx

```diff
@@ -0,0 +1,51 @@
+import * as Yup from 'yup'
+
+import { Button, Container } from '@material-ui/core'
+import { Form, Formik } from 'formik'
+
+import { ProfileSection } from './ProfileSection/ProfileSection'
+import React from 'react'
+
+export interface IUser {
+  name: string,
+  email: string,
+  dateOfBirth: string,
+  organization: string,
+  password: string
+}
+
+export interface IPasswordSettings {
+  password: string,
+  passwordConfirmation: string
```

Comment:
is this required here?

Reply:
We have the same interfaces in multiple places, maybe move them in a common place?